core/src/main/java/edu/wpi/grip/core/operations/CVOperations.java

```diff
@@ -41,6 +41,7 @@
   private final ImmutableList<OperationMetaData> imgprocOperation;
 
   @Inject
+  @SuppressWarnings({"PMD.ExcessiveMethodLength", "PMD.NcssCount"})
```

Comment:
NcssCount. 🤣

Reply:
Yeah... I don't think there's a good way to split this up, since there's so many operations that need to be provided by this class